tests/Application.Tests/ExecutorTests.cs

```diff
@@ -52,7 +52,7 @@ private class TestData : TheoryData<string, bool, bool, bool>
         {
             protected TestData(string status)
             {
-                foreach (var directory in Directory.EnumerateDirectories(Path.Combine("..", "..", "..", "..", status)))
+                foreach (var directory in Directory.EnumerateDirectories(Path.Combine("..", "..", "..", "..", "..", status)))
```

Comment:
The working path gets an extra segment when you specify the RID like above so hopefully this can be reverted too if that gets sorted.